api/src/controllers/changes.js

```diff
@@ -477,12 +516,19 @@ module.exports = {
       if (auth.isAdmin(userCtx)) {
         proxy.web(req, res);
       } else {
-        var feed = {
+        const feed = {
+          id: uuid(),
           req: req,
           res: res,
-          userCtx: userCtx
+          userCtx: userCtx,
+          upstreamSessions: {},
         };
+        feed.log = (...message) => console.log(`[feed:${feed.id}]`, ...message);
```

Comment:
Instead of doing string concatenation every time we log, set the feed.id to "\`[feed:${feed.id}]\`"? It's only used here.